### PR TITLE
vk: push back border color to sampler

### PR DESCRIPTION
**Pull request by raven02**
fix https://github.com/RPCS3/rpcs3/issues/1826

Tested with Arkedo 03 and Blade Arcus From Shining EX

### CODE REVIEW

**Comment by kd-11** [comment]
@raven02 Do we have proof that border color can only be 0 and 255 only? Coz if not, we should probably set up a default instead of failing. In my opinion, texture sampling should not fail because someone set a different/wrong border color.

**Comment by raven02** [comment]
I'm searching the PS3 SDK document regarding this border color but looks like no solid information about this . If set this default back to VK_BORDER_COLOR_FLOAT_OPAQUE_WHITE, looks okay to you? (which is in line with DX12 one)

**Comment by kd-11** [comment]
@raven02 Opaque white should be okay.

**Comment by raven02** [comment]
Thanks .I'm closing this then.

File: rpcs3/Emu/RSX/VK/VKFormats.cpp [hunk 1]
```diff
@@ -65,6 +65,16 @@ VkFilter get_mag_filter(rsx::texture_magnify_filter mag_filter)
 	throw EXCEPTION("Invalid mag filter (0x%x)", mag_filter);
 }
 
+VkBorderColor get_border_color(u8 color)
+{
+	switch (color)
+	{
+	case 0: return VK_BORDER_COLOR_FLOAT_OPAQUE_BLACK;
+	case 0xff: return VK_BORDER_COLOR_FLOAT_OPAQUE_WHITE;
```
Comment (kd-11) [inline rpcs3/Emu/RSX/VK/VKFormats.cpp]: No need to close this; it should be okay and is actually better than having a one-size-fits-all. DX12 does set border color properly by the way. Just add a default case here that returns white and you're good to go.

**Pull request closed without merging**